### DISCUSSION TITLE
Long live Hookstate!

### DISCUSSION THREAD

**Discussion by octadecimal** [avkonst/react-use-state-x#349] [General]
I will gladly take the first post to congratulate and thank you for your incredible work on Hookstate, Andrey!

It's a wonderful library that has completely changed the way I manage local state in React applications. The simplicity you've introduced has saved so many hours of boilerplate, tedium, and headache.

I also sincerely appreciate how committed you've been to this library, consistently making fixes and introducing improvements for years now.

Fun aside: I lead a team responsible for developing a rather sophisticated game editor (think Unity) that uses Hookstate for all state management. You'd probably get a kick out of seeing it - I would love to show it to you some time!

Again, a huge thank you for the incredible library that is Hookstate!

**Comment by avkonst** [comment]
Ohh.. Thank you for nice words and feedback. This is what motivates me to work on the project.
Yes, I would be glad to see what you built with Hookstate. Please contact me by avkonst@gmail.com and we have a chat. I am in NZ timezone and generally available in the late evenings during working days.
Maybe this discussion could start accumulating references to projects built with Hookstate :) That would be super interesting to see.

**Comment by onattech** [comment]
Hi, great job with hookstate. [Dataplane](https://github.com/dataplane-app/dataplane) uses hookstate for its complex UI state. Not having to deal with many nested context providers and being able to import a global state at any level without much boilerplate is great. Thank you for all the hard work.
